src/Tribe/Admin/Help_Page.php

```diff
@@ -0,0 +1,448 @@
+	 */
+	private function get_plugin_api_data( $plugin = null ) {
+		if ( is_scalar( $plugin ) ){
+			return false;
+		}
+
+		$plugin = (object) $plugin;
+		if ( empty( $plugin->api ) ){
+			return false;
+		}
+
+		/**
+		 * Filter the amount of time (seconds) we will keep api data to avoid too many external calls
+		 * @var int
+		 */
+		$timeout = apply_filters( 'tribe_help_api_data_timeout', 3 * HOUR_IN_SECONDS );
+		$transient = 'tribe_help_api_data-' . $plugin->name;
+		$data = get_transient( $transient );
+
+		if ( false === $data ) {
+			if ( ! function_exists( 'plugins_api' ) ) {
+				include_once ABSPATH . '/wp-admin/includes/plugin-install.php';
+			}
+
+			// Fetch the data
+			$data = plugins_api( 'plugin_information', array(
+				'slug' => $plugin->name,
+				'is_ssl' => is_ssl(),
+				'fields' => array(
+					'banners' => true,
+					'reviews' => true,
+					'downloaded' => true,
+					'active_installs' => true,
+				)
+			) );
+
+			if ( ! is_wp_error( $data ) ) {
+				// Format Downloaded Infomation
+				$data->downloaded = $data->downloaded ? number_format( $data->downloaded ) : _x( 'n/a', 'not available', 'tribe-common' );
+			} else {
+				// If there was a bug on the Current Request just leave
+				return false;
+			}
+
+			set_transient( $transient, $data, $timeout );
+		}
+		$data->up_to_date = ( version_compare( $plugin->version, $data->version, '<' ) ) ? esc_html__( 'You need to upgrade!', 'tribe-common' ) : esc_html__( 'You are up to date!', 'tribe-common' );
+
+		/**
+		 * Filters the API data that was stored in the Transient option
+		 *
+		 * @var array
+		 * @var object The plugin object, check `$this->get_plugins()` for more info
+		 */
+		return (object) apply_filters( 'tribe_help_api_data', $data, $plugin );
+	}
+
+	/**
+	 * Prints the Plugin box for the given plugin
+	 *
+	 * @param  string $plugin Plugin Name key
+	 * @return void
+	 */
+	public function print_plugin_box( $plugin ) {
+		$plugin = (object) $this->get_plugins( $plugin, false );
+		$api_data = $this->get_plugin_api_data( $plugin );
+		$addons = $this->get_addons( $plugin->name );
+
+		if ( $api_data ) {
+			if ( ! function_exists( 'install_plugin_install_status' ) ) {
+				include_once ABSPATH . '/wp-admin/includes/plugin-install.php';
+			}
+			$status = install_plugin_install_status( $api_data );
+			$plugin_active = is_plugin_active( $status['file'] );
+
+			if ( 'install' !== $status['status'] && ! $plugin_active ){
+				$args = array(
+					'action' => 'activate',
+					'plugin' => $status['file'],
+					'plugin_status' => 'all',
+					'paged' => 1,
+					's' => '',
+				);
+				$activate_url = wp_nonce_url( add_query_arg( $args, 'plugins.php' ), 'activate-plugin_' . $status['file'] );
+				$link = '<a class="button" href="' . $activate_url . '" aria-label="' . esc_attr( sprintf( __( 'Activate %s', 'tribe-common' ), $plugin->name ) ) . '">' . esc_html__( 'Activate Plugin', 'tribe-common' ) . '</a>';
```

Comment:
This was on WordPress core Code, just copied. I think it's something for screen readers.